style.css

```diff
@@ -0,0 +1,504 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.flex {
+  display: flex;
+}
+
+.grid {
+  display: grid;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+nav {
+  position: fixed;
+  width: 100%;
+  background-color: white;
+  z-index: 1;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+}
+
+.float-left {
+  width: 100%;
+}
+
+.nav-1 {
+  justify-content: space-between;
+  height: 30px;
+  align-items: center;
+  padding-left: 2%;
+  padding-right: 2%;
+  background-color: #131313;
+}
+
+.nav-1 a {
+  color: grey;
+  padding-right: 8px;
+  font-size: 2vh;
+}
+
+.ftr-links a {
+  margin-left: 5%;
+  margin-right: 5%;
+  color: #e1e1e1;
+}
+
+.lists a {
+  color: gray;
+}
+
+.section-3-top a {
+  padding-left: 1vw;
+  padding-right: 1vw;
+  color: #f42;
+}
+
+.nav-2-left a {
+  color: gray;
+  margin-right: 10px;
+}
+
+.nav-3 a {
+  color: black;
+}
+
+.nav-1 a:hover {
+  color: #efefef;
+}
+
+.nav-2 {
+  justify-content: space-between;
+  align-items: center;
+  height: 13vh;
+  padding-left: 2%;
+  background-color: white;
+}
+
+.nav-2-right {
+  color: red;
+}
+
+.nav-2-items {
+  margin-right: 20px;
+}
+
+.fa-search {
+  color: grey;
+}
+
+.nav-2-btn {
+  margin-left: 20px;
+  width: 100px;
+  height: 30px;
+  background-color: lightskyblue;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  font-size: 10px;
+  font-weight: bold;
+}
+
+.nav-2-btn:hover {
+  cursor: pointer;
+  background-color: #90c;
+}
+
+.nav-2-right img {
+  display: inline;
+}
+
+.hamberger {
+  visibility: hidden;
+  color: gray;
+}
+
+.nav-3 {
+  background-color: #fd3;
+  padding-left: 2%;
+  height: 5vh;
+  align-items: center;
+  width: 100%;
+}
+
+.section-1 {
+  padding: 9% 0.3% 0.3%;
+}
+
+.sec-1-right {
+  flex-direction: column;
+  margin-left: 1%;
+}
+
+.lower-flex {
+  margin-top: 1%;
+}
+
+.section-2 {
+  margin: 2% 6% 7%;
+  padding-bottom: 5%;
+  font-size: 90%;
+}
+
+.sec-2-grid {
+  padding-top: 3%;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  grid-column-gap: 2%;
+  grid-row-gap: 8%;
+  height: auto;
+}
+
+.latest-news {
+  color: #fd3;
+  font-weight: 900;
+}
+
+.latest-news:hover {
+  color: black;
+}
+
+.grid1 h3 {
+  width: 95%;
+}
+
+.section-3 {
+  margin: 7% 8%;
+  max-width: 100%;
+  margin-top: 5%;
+}
+
+.section-3-top {
+  justify-content: space-between;
+}
+
+.section-3-top-a {
+  border-right-style: solid;
+  border-left-style: solid;
+  border-right-color: #f42;
+  border-left-color: #f42;
+  border-left-width: 1px;
+  border-right-width: 1px;
+}
+
+.section-3-bottom {
+  margin-top: 5%;
+}
+
+.card {
+  flex-direction: column;
+  background-color: #fffaf8;
+  margin-right: 2%;
+}
+
+.card hr {
+  color: #f42;
+  margin-top: 5%;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your ` color: #f42;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.color-orange{
   color: #f42;
}
```
**Html file**

```HTML
<div class="color-orange">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_